### ISSUE TITLE
Define the semantic of the event handler.

### ISSUE THREAD

**Issue by gallandarakhneorg** [sarl/sarl#339]
All the event handlers must be run in parallel, even inside super types.
```
agent A1 {
   on Initialize {
       doSometing
   }
   on Initialize {
      doSometingLong
   }
}
agent A2 extends A1 {
   on Initialize {
       doSometing2
   }
}
```

In the previous example, the three handlers must be run.

**Comment by gallandarakhneorg** [comment]
Let consider the following example:
```
agent A1 {
   var x
   on Initialize {
      this.x = doSometing2
   }
   on Initialize {
      doSometing2
   }
}
agent A2 exdtends A1 {
   on Initialize {
       if (this.x == 45) {
           doSomething3
       } else {
           doSomething4
       }
   }
}
```

The code is written by assuming that the event handlers of the super type are run before the event handlers of the sub type. It is similar to the OOP.

Is SARL using this semantic?

If not, how can we code initializers that are waiting for super-type's initialization?

**Comment by gallandarakhneorg** [comment]
Fixed by janus-project/janusproject#116
